gulpfile.js

```diff
@@ -33,18 +33,17 @@ var gulp              =   require('gulp'),
 function styles(env) {
   var s = gulp.src(sassPaths);
   var isDev = env === 'dev';
-  console.log("argv env param:",isDev);
   if (isDev)
     s = s
       .pipe(sourcemaps.init());
     s = s
-    .pipe(sass().on('error', sass.logError))
     .pipe(sassGlob())
+    .pipe(sass().on('error', sass.logError))
     .pipe(bless())
     .pipe(cleanCSSMinify())
-    .pipe(sassLint())
-    .pipe(sassLint.format())
-    .pipe(sassLint.failOnError())
+    //.pipe(sassLint())
+    //.pipe(sassLint.format())
+    //.pipe(sassLint.failOnError())
```

Comment:
If we no longer want these lines then let's remove them instead of commenting them out.